### DISCUSSION TITLE
Disable Auto-edit Clipboard Text

### DISCUSSION THREAD

**Discussion by protean-onion** [TriliumNext/Trilium#2240] [Q&A]
Hello,
I've been using Trilium a lot lately. One thing I use it as is a repository of quotes that I found important in the books that I am reading. When I do this, however, a newline character is prepended to the copied text. Removing it repeatedly can get quite tedious. Could you please suggest me ways through which I can disable this so that the text is pasted as it is, without any prependments?
Thanks!

**Comment by abitofevrything** [comment]
I'm pretty sure that the newline is only appended because it's part of the selected text - triple-clicking to select the entire paragraph generally also copies the newline at the end of it.
